azure-pipelines.yml

```diff
@@ -9,7 +9,6 @@ jobs:
     variables:
       pythonVersion: '3.6'
       cmakeBuildType: Release
-      customInstallation: 'brew update && brew install graphviz doxygen boost eigen gmp mpfr tbb cgal'
```

Comment:
Does graphviz come as a dependency of something else, or is it actually unnecessary? (I still see it in the dockerfiles)